src/java.xml/share/classes/com/sun/org/apache/xml/internal/serializer/ToHTMLStream.java

```diff
@@ -1451,22 +1451,24 @@ else if ((ch < m_maxCharacter) && (m_maxCharacter == 0xFFFF)
                     {
                     writer.write(ch);  // no escaping in this case
                     }
-                    else
```

Comment:
I was suggesting removing the entire comment-out block if it is not needed (and confusing), but I will defer the decision to Joe.

Reply:
I agree. It's very obsolete. The comment-out block from line 1445 to 1454 can be removed.

Reply:
I was mistaken. I deleted the entire comment.